client/src/components/Error/Error.js

```diff
@@ -1,13 +1,25 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import errorToast from '../../utils/Toasts/default';
+import './Error.css';
 
 class Error extends Component {
+    componentDidMount() {
+        var { location } = this.props;
+        // Toast the error message
+        if (location.state && location.state.message) {
+            toast(location.state.message);
```

Comment:
Toast seems unfitting because of its animations and longevity (disappears after a few seconds. If I was the user, I would want the error message to come immediately and stay there if I choose to try to understand it). Also feels unnecessary since the error message will already be displayed on the page itself.